internal/provider/resource_gitlab_project_test.go

```diff
@@ -898,8 +894,7 @@ func testAccCheckGitlabProjectMirroredAttributes(project *gitlab.Project, want *
 	}
 }
 
-// lintignore: AT002 // not a Terraform import test
-func TestAccGitlabProject_importURLMirrored(t *testing.T) {
+func TestAccGitlabProject_ImprtURLMirrored(t *testing.T) {
 	// Since we do some manual setup in this test, we need to handle the test skip first.
 	if os.Getenv(resource.EnvTfAcc) == "" {
 		t.Skip(fmt.Sprintf("Acceptance tests skipped unless env '%s' set", resource.EnvTfAcc))
```

Comment:
while you are at it, can you replace this with a `testAccCheck(t)` ?

Reply:
Done